common-content/en/module/how-our-curriculum-works/importance-of-prep/index.md

```diff
@@ -0,0 +1,11 @@
++++
+title="🧑🏾‍💻 Importance of prep"
+headless="true"
+time= 30
+[objectives]
+    1="Explain the importance of prep"
++++
+
+In a flipped learning model, learners are expected to **prepare** before they meet up as a community. Therefore, regular preparation is essential for our community to self-educate together. In each sprint week, we expect trainees to work on the prep section beforehand to start building their own mental model of the week's concepts.
+
+Here is an example of prep from JS1 to gain an example of what trainees will be expected to engage with before an in-person session: https://curriculum.codeyourfuture.io/js1/sprints/1/prep/
```

Comment:
Since we intend to have new volunteers go through this, perhaps we should try to avoid the jargon?

```suggestion
Here is an example of prep from the first JavaScript module to gain an example of what trainees will be expected to engage with before an in-person session: https://curriculum.codeyourfuture.io/js1/sprints/1/prep/
```